tests/dummy/app/templates/application.hbs

```diff
@@ -20,6 +20,22 @@
         </li>
       </ul>
     </div>
+
+    <div class="uk-navbar-right">
+      <ul class="uk-navbar-nav">
+        <li>
+          <LinkTo @query={{hash activeGroup="dev-group"}}>
+            dev-group
+          </LinkTo>
+        </li>
+
+        <li>
+          <LinkTo @query={{hash activeGroup="secondary-group"}}>
+            secondary-group
+          </LinkTo>
+        </li>
```

Comment:
This looks like prototype code. Is this intended to be in `main`?

Reply:
Both. It's the same idea as with https://github.com/projectcaluma/ember-alexandria/blob/main/tests/dummy/app/templates/application.hbs#L11-L20

Reply:
Ah sorry :sweat_smile: , i didn't see this was in the dummy app!